### PR TITLE
[default] Switch default OperatorSources to CatalogSources

### PR DESCRIPTION
**Pull request by anik120**
**Description of the change:**

This PR switches the default OperatorSources to CatalogSources.

**Motivation for the change:**

**Reviewer Checklist**
- [ ] Implementation matches the proposed design, or proposal is updated to match implementation
- [ ] Sufficient unit test coverage 
- [ ] Sufficient end-to-end test coverage
- [ ] Docs updated or added to `/docs` 
- [ ] Commit messages sensible and descriptive

### CODE REVIEW

File: defaults/03_community_operators.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: "community-operators"
   namespace: "openshift-marketplace"
-  labels:
-    opsrc-provider: community
 spec:
-  type: appregistry
-  endpoint: "https://quay.io/cnr"
-  registryNamespace: "community-operators"
+  sourceType: grpc
+  image: registry.redhat.io/redhat/community-operator-index:v4.6
```
Comment (shawn-hurley) [inline defaults/03_community_operators.yaml:8]: I believe that we are changing this:

`quay.io/openshift-community-operators/catalog:latest`

**Comment by anik120** [comment]
/hold 
Blocked by https://github.com/operator-framework/operator-lifecycle-manager/pull/1682

**Comment by anik120** [comment]
/hold cancel 

Not adding the `priority` field to CatalogSources for this PR.

**Comment by anik120** [comment]
/hold
Waiting for operator-marketplace image to become available.

File: defaults/04_redhat_marketplace.yaml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
   name: "redhat-marketplace"
   namespace: "openshift-marketplace"
-  labels:
-    opsrc-provider: marketplace
 spec:
-  type: appregistry
-  endpoint: "https://quay.io/cnr"
-  registryNamespace: "redhat-marketplace"
+  sourceType: grpc
+  image: registry.redhat.io/redhat/redhat-marketplace-operator-index:v4.6
```
Comment (ecordell) [inline defaults/04_redhat_marketplace.yaml:8]:
```suggestion
  image: registry.redhat.io/redhat/redhat-marketplace-index:v4.6
```

**Pull request merged**